packages-exp/remote-config-exp/test_app/index.js

```diff
@@ -48,8 +48,8 @@ window.onload = function () {
 
 function initializeFirebase() {
   const val = document.querySelector('#firebase-config').value;
-  firebase.initializeApp(parseObjFromStr(val));
-  remoteConfig = firebase.remoteConfig();
+  const app = firebase.app.initializeApp(parseObjFromStr(val));
+  remoteConfig = firebase.remoteConfig.getRemoteConfig(app);
```

Comment:
@Feiyang1 since we assign RC to a global var here, do we need to prefix RC references w `firebase` below? 
If we no longer need the global var, mind removing it?

Reply:
`remoteConfig` is the remote config instance. `firebase.remoteConfig` is the namespace where rc functions are attached. We still need it and to avoid confusion, I renamed `remoteConfig` to `rcInstance`.